services/system/AuthSys/plugin/Cargo.toml

```diff
@@ -0,0 +1,21 @@
+[package]
+name = "auth-sys"
+version = "0.1.0"
+edition = "2021"
+
+[dependencies]
+bitflags = "2.4.2"
+wit-bindgen-rt = "0.22.0"
+#psibase = { version = "0.7.0", path = "../../../../rust/psibase/" }
```

Comment:
Cruft? or left in here for some future use?

Reply:
Cruft, removed